apps/studio/components/layouts/TableEditorLayout/EntityListItem.tsx

```diff
@@ -314,6 +326,17 @@ const EntityListItem: ItemRenderer<Entity, EntityListItemProps> = ({
               <IconDownload size="tiny" />
               <span>Export as CSV</span>
             </DropdownMenuItem>
+            <DropdownMenuItem
+              key="copy-definition"
+              className="space-x-2"
+              onClick={(e) => {
+                e.stopPropagation()
+                copyDefinition()
+              }}
+            >
+              <IconCopy size="tiny" />
```

Comment:
I think let's use https://lucide.dev/icons/files instead?

Reply:
I think that icon is usable. I'll change it.

Reply:
Changed the icon to `IconClipboard`.
I've tried the files one from lucide, but it wasn't displaying in the correct color as the other icons. Also, because the function copy's the definition to the clipboard, I thought maybe this one was more suiting. Let me know.